man/man8/tcprtt.8

```diff
@@ -2,7 +2,7 @@
 .SH NAME
 tcprtt \- Trace TCP RTT of established connections. Uses Linux eBPF/bcc.
 .SH SYNOPSIS
-.B tcprtt [\-h] [\-T] [\-D] [\-m] [\-i INTERVAL] [\-d DURATION] [\-b] [\-B]
+.B tcprtt [\-h] [\-T] [\-D] [\-m] [\-i INTERVAL] [\-d DURATION] [\-b] [\-B] [\-v]
```

Comment:
Let us sue `-e` (extension) instead of `-v`. The `-e` is used by biolatency.py to show additional stats. In the future, more things can be added under `-e` if needed.